src/items/static-items.md

```diff
@@ -22,11 +22,8 @@ Static initializers may refer to other statics.
 Non-`mut` static items that contain a type that is not [interior mutable] may
 be placed in read-only memory.
 
-All access to a static is safe, but there are a number of restrictions on
-statics:
-
-* The type must have the `Sync` trait bound to allow thread-safe access.
-* Constants cannot refer to statics.
+All access to a static is safe,
+provided that the type must have the `Sync` trait bound to allow thread-safe access.
```

Comment:
This makes it sound like a `!Sync` static would be unsafe to access. But actually it's just rejected.

Reply:
Thanks! I have rewritten this to use more direct language. I'm not sure why the original was written to say "it is safe to ...", since there isn't anything to imply that it was unsafe.

Reply:
I fixed the wording, to mention that a type with `!Sync` is rejected.

Reply:
I picked the changeset from @ehuss actually. I think those wordings are better.